kore/src/Kore/Logger/Output.hs

```diff
@@ -177,7 +172,7 @@ parseKoreLogOptions =
             "warning"  -> pure Warning
             "error"    -> pure Error
             "critical" -> pure Critical
-            _          -> Nothing
+            _          -> pure Warning
```

Comment:
If the user supplies `--log-level`, but we don't recognize the string, it should be a parse failure.

```suggestion
            _          -> Nothing
```